### PR TITLE
test files under stdlib/2and3 with both python versions

### PR DESCRIPTION
**Pull request by martindemello**

### CODE REVIEW

File: tests/pytype_test.py [hunk 1]
```diff
@@ -174,6 +174,21 @@ def pytype_test(args):
     pytd_run = []
     bad = []
 
+    def _make_test(filename, major_version):
+        run_cmd = [
+            pytype_exe,
+            '--module-name=%s' % _get_module_name(filename),
+            '--parse-pyi'
```
Comment (gvanrossum) [inline tests/pytype_test.py]: Personally I'd use a trailing comma here.

File: tests/pytype_test.py [hunk 2]
```diff
@@ -187,31 +202,28 @@ def pytype_test(args):
     running_tests = collections.deque()
     max_code, runs, errors = 0, 0, 0
     files = pytype_run + pytd_run
-    total_files = len(files)
+    total_tests = len(files)
+    # Files in stdlib/2and3 get tested twice
+    total_tests += len([f for f in pytype_run if 'stdlib/2and3' in f])
```
Comment (rchen152) [inline tests/pytype_test.py]: You can do this in a slightly more concise way as sum(1 for f in ...).

File: tests/pytype_test.py (hunk 1 above)
Comment (martindemello) [reply tests/pytype_test.py]: done

File: tests/pytype_test.py (hunk 2 above)
Comment (martindemello) [reply tests/pytype_test.py]: done

**Pull request merged**